src/status_im/events.cljs

```diff
@@ -1997,4 +2002,16 @@
 (re-frame/reg-fx
  :dismiss-keyboard
  (fn []
-   (react/dismiss-keyboard!)))
\ No newline at end of file
+   (react/dismiss-keyboard!)))
+
+(handlers/register-handler-fx
+ :wallet-send-request
+ (fn [{:keys [db] :as cofx} [_ public-key amount symbol decimals]]
+   (assert public-key)
+   (let [request-command (get-in db [:id->command ["request" #{:personal-chats}]])]
+     (fx/merge cofx
+               (chat/start-chat public-key nil)
```

Comment:
Why that?

Reply:
i just moved this code, i think we want to send a message and . if there is no chat we want to start it